### DISCUSSION TITLE
postrgesql connection not working after switching to CLI

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#6076] [Help] [answered]
I set up the budibase cli for running my self-host, and now my connections to my postgresql database (Which is not in Docker) no longer work. I tried changing the host to host.docker.internal, localhost. and the ip of the machine, none of which work.

**Comment by melohagan** [comment, marked as answer]
Hi @gondilon 

What errors do you see in the network tab? What version is your app?

**Reply by snvgglebear** [reply]
This is resolved now after restarting Postgres and updating the pg_hba.conf file with the correct IP for the docker container.
